daemon/graphdriver/overlay2/overlay.go

```diff
@@ -213,7 +216,12 @@ func parseOptions(options []string) (*overlayOptions, error) {
 			if err != nil {
 				return nil, err
 			}
-
+		case "overlay2.default_size":
```

Comment:
Because the storage option name in create/start command is the same for all graphdrivers that support it (i.e. "size=") I think the option name for dockerd storage option should be equally generic (i.e. "default_size="). This will also make documentation more generic. BTW, how about the documentation?